cs-offerings/scripts/delete/delete_chaincode-instantiate.sh

```diff
@@ -21,12 +21,12 @@ echo "Preparing yaml for chaincode instantiate delete"
 sed -e "s/%CHANNEL_NAME%/${CHANNEL_NAME}/g" -e "s/%PEER_ADDRESS%/${PEER_ADDRESS}/g" -e "s/%PEER_MSPID%/${PEER_MSPID}/g" -e "s|%MSP_CONFIGPATH%|${MSP_CONFIGPATH}|g"  -e "s/%CHAINCODE_NAME%/${CHAINCODE_NAME}/g" -e "s/%CHAINCODE_VERSION%/${CHAINCODE_VERSION}/g" ${KUBECONFIG_FOLDER}/chaincode_instantiate.yaml.base > ${KUBECONFIG_FOLDER}/chaincode_instantiate.yaml
 
 echo "Deleting Existing Instantiate Chaincode Pod"
-if [ "$(kubectl get pods | grep chaincodeinstantiate | wc -l | awk '{print $1}')" != "0" ]; then
+if [ "$(kubectl get pods chaincodeinstantiate | grep chaincodeinstantiate | wc -l | awk '{print $1}')" != "0" ]; then
```

Comment:
Why is this change required?

Reply:
I kept it consistent with the version that was committed in the master branch by rddill. I also observed that $(kubectl get pods | grep chaincodeinstantiate wasn't working. I had to use $(kubectl get pods -a | grep chaincodeinstantiate or $(kubectl get pods chaincodeinstantiate for this to work

Reply:
thanks